Libraries/Components/AppleTV/TVEventHandler.android.js

```diff
@@ -11,10 +11,35 @@
  */
 'use strict';
 
-function TVEventHandler() {}
+const React = require('React');
```

Comment:
Can we consolidate TVEventHandler.android.js and TVEventHandler.ios.js? From what I understand the only difference is that the native module we pass to NativeEventEmitter for iOS does not exists on android (in which case its undefined which seems to be ok with this code)